src/options/options.js

```diff
@@ -0,0 +1,9 @@
+import Options from './Options.svelte';
```

Comment:
it should be _options.svelte_ instead of _Options.svelte_

Reply:
Hi @jatinhemnani01,
thanks for your awesome project. Do you have a roadmap?
It is strange that the filename is in lower case. This line is the same like './App.svelte'. I will try to camelcase the filename.

Reply:
it's ok, after done coding you can _npm run build_ to check if everything is working fine
Thanks for your contribution.